src/test/java/nextstep/subway/acceptance/SectionAcceptanceTest.java

```diff
@@ -61,6 +62,7 @@ void createSection() {
         응답코드_검증(response, HttpStatus.OK);
         LineResponse line7 = 지하철_노선_조회_응답값_반환(1L);
         상행_종점역_기대값_검증(line7, 첫번째역, "첫번째역");
+        역_위치_기대값_검증(line7, 두번째역, "두번째역", 1);
```

Comment:
index 값을 활용해서 검증하셨네요 😃 
전체 역 목록을 검증하면 테스트 가독성이 높아질 것 같은데 어떻게 생각하세요?
```suggestion
        assertThat(역_목록).containsExactly(첫번째역, 두번째역, 세번째역);
```